src/lib/isSemVer.js

```diff
@@ -0,0 +1,33 @@
+import assertString from './util/assertString';
+
+/**
+ * Build RegExp object from an array
+ * of multiple/multi-line regexp parts
+ *
+ * @param {string[]} parts
+ * @param {string} flags
+ * @return {object} - RegExp object
+ */
+function buildMultiLineRegexp(parts, flags = '') {
```

Comment:
I like this -- really improves readability. Perhaps we should move it to the utilities to be re-used later? in `/src/lib/util/multilineRegex.js`

Reply:
Yeah, totally agree! The main reason behind breaking the **LONG** regular expression into multi-parts/lines is to offer better readability 👍 .. Will move it into `util` folder to be re-usable